README.md

```diff
@@ -94,25 +114,25 @@ Before you start to implement, you will want to become familiar with a few data
 | Name             | Description                                                  |
 | ---------------- | ------------------------------------------------------------ |
 | PSSignerID       | `PSSignerID` is a typealias for String.                      |
-| PSSigner         | `PSSigner` is a structure that you'll use to send over your signer information. You must include a signer ID (`PSSignerID`) when needing to send data to PactSafe. You can optionally pass over additional custom data with a `PSCustomData` struct, which is covered below. |
+| PSSigner         | `PSSigner` is a structure that you'll use to send over your signer information. You must include a signer ID (`PSSignerID`) when needing to send data to Ironclad Clickwrap. You can optionally pass over additional custom data with a `PSCustomData` struct, which is covered below. |
 | PSCustomData     | `PSCustomData` is a structure that holds additional information about the activity. Please see the properties that are available to be set in the [Customizing Acceptance Data](#customizing-acceptance-data) section. |
-| PSGroup          | `PSGroup` is a structure that holds information about a speciifc group (uses PactSafe group key) that is loaded from the PactSafe API. |
-| PSContract       | `PSContract` is a structure that holds information about contracts within a PactSafe group (`PSGroup`). |
+| PSGroup          | `PSGroup` is a structure that holds information about a speciifc group (uses Ironclad Clickwrap group key) that is loaded from the PactSafe API. |
+| PSContract       | `PSContract` is a structure that holds information about contracts within a Ironclad Clickwrap group (`PSGroup`). |
 | PSConnectionData | The `PSConnectionData` structure holds information about the current connection [Customizing Acceptance Data](#customizing-acceptance-data) section. |
 
 
 
-## Configure and Initalize the PactSafe SDK
+## Configure and Initialize the Ironclad Clickwrap SDK
 
-In order to use the PactSafe SDK, you’ll need to import PactSafe into your UIApplicationDelegate:
+In order to use the Ironclad Clickwrap SDK, you’ll need to import PactSafe into your UIApplicationDelegate:
```

Comment:
Same note as leaving PactSafe here on line 127. Let me know if this needs updated too